api/go.mod

```diff
@@ -6,16 +6,17 @@ require (
 	github.com/blang/semver/v4 v4.0.0
 	github.com/go-errors/errors v1.4.2
 	github.com/google/shlex v0.0.0-20191202100458-e7afc7fbc510
-	github.com/stretchr/testify v1.8.1
+	github.com/stretchr/testify v1.8.4
 	go.uber.org/goleak v1.3.0
 	gopkg.in/evanphx/json-patch.v4 v4.12.0
+	k8s.io/klog v1.0.0
 	k8s.io/kube-openapi v0.0.0-20230601164746-7562a1006961
 	sigs.k8s.io/kustomize/kyaml v0.16.0
 	sigs.k8s.io/yaml v1.4.0
 )
 
 require (
-	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc // indirect
```

Comment:
Why were these updated to non-released versions? They're not even a dependency go protobuf?
https://github.com/protocolbuffers/protobuf-go/blob/v1.33.0/go.mod#L5-L10

This caused a massive ripple effect in any project having a dependency on kustomize or kubernetes (direct or indirect) 😞